src/platform/user/authentication/utilities.js

```diff
@@ -99,6 +103,10 @@ export function logout(version = 'v0') {
   return redirect(sessionTypeUrl('slo', version), 'logout-link-clicked');
 }
 
+export function autoLogout() {
+  return redirect(sessionTypeUrl('slo', 'v1'), 'sso-automatic-logout');
```

Comment:
Again as someone that knows nothing, what does `'slo'` stand for?

Reply:
I actually had to double check this, but thankfully it actually makes sense and it's Single Log Out to our Single Sign On 🙂